ChannelApe/generator-channelape
# Adding the web services SDK to the yeoman generator

The major update to this repo is the moving the abstract ChannelApeActions controller out of the templates in order to utilize the version found in the common web services sdk.

A unit test was also added in order to ensure the proper handling of --ca and -o and -s flags.

To test (without publishing to NPM):

Pull **feature/web-services-sdk**
Run **npm i**
Run **npm i -g**
In a terminal run **yo channelape {session-id} --ca -o -s**

Once the vscode window opens you can peruse the repo at your will. If you run the *LAUNCH* configurations you can install **http://{your-local-ip}:3000/orders** on a test business to see that it grabs all open orders and iterates over them.
